csharp/AppEncryption/AppEncryption/Persistence/AdoMetastoreImpl.cs

```diff
@@ -171,6 +171,11 @@ public bool Store(string keyId, DateTimeOffset created, JObject value)
             }
         }
 
+        public string GetKeySuffix()
+        {
+            return string.Empty;
```

Comment:
Looks like we're missing a unit test here (thanks codecov)

Reply:
Ah, yup, forgot I had to add some net new stuff in here. :) Good catch!